src/therapy/etl/hemonc.py

```diff
@@ -206,11 +197,42 @@ def _get_synonyms(self, therapies: dict) -> dict:
                         therapies[therapy_code]["aliases"].append(row[0])
             return therapies
 
+    def _perform_qc(self, therapies: dict[str, dict]) -> dict[str, dict]:
+        """Perform HemOnc-specific QC checks on therapy records.
+
+        Indications that a record is a combo therapy:
+        * has multiple RxNorm xrefs
+        * has " and " in label
+
+        :param therapies: collection of records from HemOnc
+        :return: collection w/o failing records
+        """
```

Comment:
is this problem always hemonc specific? would there be anywhere else this qc function would be useful?

Reply:
yeah, good q. There's a separate section of filter/checking in the parent class but these checks are rough enough that I wanted to limit their applicability (hemonc is small enough that you can just check manually). If we see this problem from other sources we could definitely migrate it up to the parent.